### PR TITLE
DOCS_8060 ASM SCA env var enablement

### PR DESCRIPTION
**Pull request by michaelcretzman**
### What does this PR do? What is the motivation?

adds new topic on Setting up ASM SCA using env var

### Merge instructions

- [ ] Please merge after reviewing

### Additional notes

Needs PM review

### CODE REVIEW

File: content/en/security/application_security/enabling/_index.md [hunk 1]
```diff
@@ -33,13 +33,26 @@ further_reading:
 <div class="alert alert-warning">Application Security Management is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
 {{< /site-region >}}
 
-Enable your application to [detect and protect against threats][1] targeting your production systems, and to [manage risks][2] in your code and its open source dependencies, using the Datadog library for your application language. You can detect vulnerabilities and threats for apps hosted on a server, Docker, Kubernetes, Amazon ECS, and (for supported languages) AWS Fargate.
+You can use Datadog Application Security Management products to: 
+
+- Enable your application to [detect and protect against threats][1] targeting your production systems. 
+- [Manage risks][2] in your code and its open source dependencies using the Datadog library for your application language. 
+- Detect vulnerabilities and threats for apps hosted on a server, Docker, Kubernetes, Amazon ECS, and (for supported languages) AWS Fargate.
```
Comment (cswatt) [inline content/en/security/application_security/enabling/_index.md:40]:
```suggestion
- Detect vulnerabilities and threats for apps hosted on a server, Docker, Kubernetes, Amazon ECS, or (for supported languages) AWS Fargate.
```

my one deeply pedantic edit

**Review by cswatt — APPROVED** [review]

File: content/en/security/application_security/enabling/tracing_libraries/_index.md [hunk 2]
```diff
@@ -14,6 +14,8 @@ The following documentation provides instructions to enable ASM using Datadog tr
     {{< nextlink href="/security/application_security/enabling/tracing_libraries/threat_detection" >}}Enabling ASM for Threat Detection and Protection using Datadog tracing libraries{{< /nextlink >}}
 
      {{< nextlink href="/security/application_security/enabling/tracing_libraries/code_security" >}}Enabling ASM for Code Security using Datadog tracing libraries{{< /nextlink >}}
+
+     {{< nextlink href="/security/application_security/enabling/tracing_libraries/sca" >}}Enabling ASM for Software Composition Analysis using Datadog tracing libraries{{< /nextlink >}}
```
Comment (michaelcretzman) [inline content/en/security/application_security/enabling/tracing_libraries/_index.md:18]:
```suggestion
    {{< nextlink href="/security/application_security/enabling/tracing_libraries/threat_detection" >}}Enabling Threat Detection and Protection using Datadog tracing libraries{{< /nextlink >}}

     {{< nextlink href="/security/application_security/enabling/tracing_libraries/code_security" >}}Enabling Code Security using Datadog tracing libraries{{< /nextlink >}}

     {{< nextlink href="/security/application_security/enabling/tracing_libraries/sca" >}}Enabling Software Composition Analysis using Datadog tracing libraries{{< /nextlink >}}
```

File: content/en/security/application_security/enabling/tracing_libraries/sca/_index.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
+---
+title: Enabling ASM for Software Composition Analysis using Datadog tracing libraries
```
Comment (michaelcretzman) [inline content/en/security/application_security/enabling/tracing_libraries/sca/_index.md:2]:
```suggestion
title: Enabling Software Composition Analysis using Datadog tracing libraries
```

File: content/en/security/application_security/enabling/tracing_libraries/sca/_index.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+  text: "Troubleshooting Application Security Management"
+- link: "/security/application_security/how-appsec-works/"
+  tag: "Documentation"
+  text: "How Application Security Management Works in Datadog"
+- link: "https://www.datadoghq.com/blog/secure-serverless-applications-with-datadog-asm/"
+  tag: "Blog"
+  text: "Secure serverless applications with Datadog ASM"
+---
+
+You can use Datadog Software Composition Analysis (SCA) to monitor the open source libraries in your apps for vulnerabilities.
```
Comment (michaelcretzman) [inline content/en/security/application_security/enabling/tracing_libraries/sca/_index.md:20]:
```suggestion
You can use Datadog Software Composition Analysis (SCA) to monitor the open source libraries in your apps.
```

File: content/en/security/application_security/enabling/tracing_libraries/sca/_index.md [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+
+SCA is configured by setting the `-Ddd.appsec.sca.enabled` flag or the `DD_APPSEC_SCA_ENABLED` environment variable to `true` in supported languages.
+
+This topic explains how to set up SCA using a Java example.
+
+{{% appsec-getstarted %}}
+
```
Comment (michaelcretzman) [inline content/en/security/application_security/enabling/tracing_libraries/sca/_index.md:27]:
```suggestion



```

**Review by michaelcretzman — COMMENTED** [review]
incorporating PM review

File: content/en/security/application_security/enabling/tracing_libraries/sca/_index.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+  tag: "Documentation"
+  text: "How Application Security Management Works in Datadog"
+- link: "https://www.datadoghq.com/blog/secure-serverless-applications-with-datadog-asm/"
+  tag: "Blog"
+  text: "Secure serverless applications with Datadog ASM"
+---
+
+You can use Datadog Software Composition Analysis (SCA) to monitor the open source libraries in your apps.
+
+SCA is configured by setting the `-Ddd.appsec.sca.enabled` flag or the `DD_APPSEC_SCA_ENABLED` environment variable to `true` in supported languages.
```
Comment (michaelcretzman) [inline content/en/security/application_security/enabling/tracing_libraries/sca/_index.md:22]:
```suggestion
SCA is configured by setting the `-Ddd.appsec.sca.enabled` flag or the `DD_APPSEC_SCA_ENABLED` environment variable to `true` in supported languages:

- Java
- .NET
- Go
- Ruby
- PHP
- Node.js
- Python
```

**Comment by michaelcretzman** [comment]
this was reviewed and approved by PM.

**Pull request merged**